### PR TITLE
fix(Input): style bug

### LINKED ISSUE #41125: Input 组件 TextArea添加删除按钮， 文字太多会遮挡住删除按钮
### Reproduction link

[https://ant.design/components/input-cn#inputtextarea](https://ant.design/components/input-cn#inputtextarea)

image

### Steps to reproduce

官网demo即可复现

### What is expected?

文字不遮挡删除按钮

### What is actually happening?

文字遮挡了

| Environment | Info |
| --- | --- |
| antd | 5.3.0 |
| React | React17/18 |
| System | Mac |
| Browser | chrom |

### PR DESCRIPTION
**Pull request by Ali-ovo**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #41125

### 💡 Background and solution

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  fix style bug         |
| 🇨🇳 Chinese |  修复文字覆盖关闭按钮的问题         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
CI failed

**Comment by Ali-ovo** [comment]
> CI failed

I don't quite understand this
image

**Comment by Ali-ovo** [comment]
> 

done

File: components/input/TextArea.tsx [hunk 1]
```diff
@@ -92,6 +92,9 @@ const TextArea = forwardRef<TextAreaRef, TextAreaProps>(
         classes={{
           affixWrapper: classNames(
             `${prefixCls}-textarea-affix-wrapper`,
+            {
+              [`${prefixCls}-textarea-show-allowClear`]: mergedAllowClear,
```
Comment (MadCcc) [inline components/input/TextArea.tsx:96]: 判断 suffix 比较好，clear icon 会放在 rc-textarea 的 suffix 下面。这个 class 可以去 rc-textarea 加。

Comment (MadCcc) [reply components/input/TextArea.tsx:96]: rc-textarea 里有是否展示 suffix 的判断，className 可以在那里添加。antd 这里只添加样式。

**Pull request closed without merging**